enhancements/network/service-egress-traffic-steering.md

```diff
@@ -0,0 +1,284 @@
+                          k8s.ovn.org/egress-service-host: "ovn-worker"                (set by ovn-k)
+Type:                     LoadBalancer
+LoadBalancer Ingress:     172.19.0.100
+Endpoints:                10.244.0.3:8080,10.244.1.6:8080
+```
+the egress traffic flow for the pod `10.244.1.6` on `ovn-worker2` towards an external destination (172.19.0.5) will look like:
+```none
+                           ┌────────────────────┐
+                           │                    │
+                           │external destination│
+                           │    172.19.0.5      │
+                           │                    │
+                           └───▲────────────────┘
+                               │
+           5. packet reaches   │                      2. router policy rereoutes it
+              the external     │                         to ovn-worker's mgmt port
+              destination with │                      ┌──────────────────┐
+              src ip:          │                  ┌───┤ovn cluster router│
+              172.19.0.100     │                  │   └───────────▲──────┘
+                               │                  │               │
+                               │                  │               │1. packet to 172.19.0.5
+                            ┌──┴───┐        ┌─────▼┐              │   heads to the cluster router
+                         ┌──┘ eth1 └──┐  ┌──┘ mgmt └──┐           │   as usual
+                         │ 172.19.0.2 │  │ 10.244.0.2 │           │
+                         ├─────▲──────┴──┴─────┬──────┤           │   ┌────────────────┐
+      4. an iptables rule│     │   ovn-worker  │3.    │           │   │  ovn-worker2   │
+         that SNATs to   │     │               │      │           │   │                │
+         the service's ip│     │               │      │           │   │                │
+         and sends it    │     │               │      │           │   │ ┌────────────┐ │
+         through eth1    │     │4.┌────────┐   │      │           └───┼─┤    pod     │ │
+         is hit          │     └──┤iptables◄───┘      │               │ │ 10.244.1.6 │ │
+                         │        └────────┘          │               │ └────────────┘ │
+                         │                            │               │                │
+                         └────────────────────────────┘               └────────────────┘
+                          3. from the mgmt port it goes to
+                             ovn-worker's iptables rules
+                             before heading out
+
+```
+As mentioned earlier, for the opposite direction (ingress/external client initiates) to work properly the LoadBalancer provider needs to announce the service only from `ovn-worker`.
+
+### Usage Example
+
+With all of the above implemented, a user can follow these steps to create a functioning LoadBalancer service whose endpoints exit the cluster with its IP.
```

Comment:
should you mention here this is a metal lb LoadBalancer?

Reply:
added